### PR TITLE
Release v0.2.0

### PR DESCRIPTION
**Pull request by wtty-fool**
## Checklist
- [x] On hold until architect-orb pr is merged
- [x] Update changelog in CHANGELOG.md.

### CODE REVIEW

**Comment by xh3b4sd** [comment]
Let's please use the architect orb as well before we release `v0.2.0`.

**Comment by wtty-fool** [comment]
@xh3b4sd Sure. I'll put this on hold.

File: CHANGELOG.md [hunk 1]
```diff
@@ -18,5 +20,6 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - First release.
 
-[Unreleased]: https://github.com/giantswarm/errors/compare/v0.1.0...HEAD
+[Unreleased]: https://github.com/giantswarm/errors/compare/v0.2.0...HEAD
+[0.2.0]: https://github.com/giantswarm/errors/releases/tag/v0.2.0
```
Comment (xh3b4sd) [inline CHANGELOG.md]: I am not sure how this was supposed to be. I started out with the tag links as well but then later used the compare links. I think there is no real right or wrong but people may have a preference on how this should be. /cc @kopiczko

Comment (wtty-fool) [reply CHANGELOG.md]: I took it from [here](https://raw.githubusercontent.com/giantswarm/gatekeeper-control-plane-rules/master/CHANGELOG.md?token=ABDABCXAET7RQXSFZWFMJDS6QSFJ6)

Comment (kopiczko) [reply CHANGELOG.md]: Source of truth should be always https://keepachangelog.com/en/1.0.0/.

See:

```
[Unreleased]: https://github.com/olivierlacan/keep-a-changelog/compare/v1.0.0...HEAD
[1.0.0]: https://github.com/olivierlacan/keep-a-changelog/compare/v0.3.0...v1.0.0
[0.3.0]: https://github.com/olivierlacan/keep-a-changelog/compare/v0.2.0...v0.3.0
[0.2.0]: https://github.com/olivierlacan/keep-a-changelog/compare/v0.1.0...v0.2.0
[0.1.0]: https://github.com/olivierlacan/keep-a-changelog/compare/v0.0.8...v0.1.0
[0.0.8]: https://github.com/olivierlacan/keep-a-changelog/compare/v0.0.7...v0.0.8
[0.0.7]: https://github.com/olivierlacan/keep-a-changelog/compare/v0.0.6...v0.0.7
[0.0.6]: https://github.com/olivierlacan/keep-a-changelog/compare/v0.0.5...v0.0.6
[0.0.5]: https://github.com/olivierlacan/keep-a-changelog/compare/v0.0.4...v0.0.5
[0.0.4]: https://github.com/olivierlacan/keep-a-changelog/compare/v0.0.3...v0.0.4
[0.0.3]: https://github.com/olivierlacan/keep-a-changelog/compare/v0.0.2...v0.0.3
[0.0.2]: https://github.com/olivierlacan/keep-a-changelog/compare/v0.0.1...v0.0.2
[0.0.1]: https://github.com/olivierlacan/keep-a-changelog/releases/tag/v0.0.1
```

**Pull request merged**